modules/tinymce/src/plugins/tabfocus/test/ts/browser/TabfocusSanityTest.ts

```diff
@@ -1,54 +1,40 @@
+import { assert } from 'chai';
 
-import TabfocusPlugin from 'tinymce/plugins/tabfocus/Plugin';
+import Editor from 'tinymce/core/api/Editor';
+import Plugin from 'tinymce/plugins/tabfocus/Plugin';
 import Theme from 'tinymce/themes/silver/Theme';
 
-UnitTest.asynctest('browser.tinymce.plugins.tabfocus.TabfocusSanityTest', (success, failure) => {
-
-  Theme();
-  TabfocusPlugin();
-
-  const sAddInputs = (editor) => {
-    return Logger.t('Add inputs', Step.sync(() => {
-      const container = editor.getContainer();
-      const input1 = document.createElement('input');
-      input1.id = 'tempinput1';
+describe('browser.tinymce.plugins.tabfocus.TabfocusSanityTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    plugins: 'tabfocus',
+    tabfocus_elements: 'tempinput1',
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Theme, Plugin ], true);
 
-      container.parentNode.insertBefore(input1, container);
-    }));
+  const addInputs = (editor: Editor) => {
+    const container = editor.getContainer();
+    const input1 = document.createElement('input');
+    input1.id = 'tempinput1';
+    container.parentNode.insertBefore(input1, container);
   };
 
-  const sRemoveInputs = Logger.t('Remove inputs', Step.sync(() => {
+  const removeInputs = () => {
     const input1 = document.getElementById('tempinput1');
-
     input1.parentNode.removeChild(input1);
-  }));
-
-  TinyLoader.setupLight((editor, onSuccess, onFailure) => {
-    const tinyActions = TinyActions(editor);
-    const tinyApis = TinyApis(editor);
+  };
 
-    Pipeline.async({},
-      Log.steps('TBA', 'TabFocus: Add an input field outside the editor, focus on the editor, press the tab key and assert focus shifts to the input field', [
-        sAddInputs(editor),
-        tinyApis.sFocus(),
-        Step.sync(() => {
-          Assert.eq('should be same', 'IFRAME', document.activeElement.nodeName);
-        }),
-        tinyActions.sContentKeystroke(Keys.tab(), {}),
-        Waiter.sTryUntil('wait for focus',
-          Step.sync(() => {
-            const input = document.getElementById('tempinput1');
-            Assert.eq('should be same', input.outerHTML, document.activeElement.outerHTML);
-          })),
-        sRemoveInputs
-      ])
-      , onSuccess, onFailure);
-  }, {
-    plugins: 'tabfocus',
-    tabfocus_elements: 'tempinput1',
-    base_url: '/project/tinymce/js/tinymce'
-  }, success, failure);
+  it('TBA: Add an input field outside the editor, focus on the editor, press the tab key and assert focus shifts to the input field', async () => {
+    const editor = hook.editor();
+    addInputs(editor);
+    assert.equal(document.activeElement.nodeName, 'IFRAME');
+    Keyboard.activeKeystroke(TinyDom.document(editor), Keys.tab());
+    await Waiter.pTryUntil('Wait for focus', () => {
+      const input = document.getElementById('tempinput1');
+      assert.equal(document.activeElement.outerHTML, input.outerHTML);
+    });
```

Comment:
```suggestion
    await FocusTools.pTryOnSelector('Wait for focus to be on input', SugarDocument.getDocument(), '#tempinput1');
```
Again, this a very old way of doing things, so this uses the modern version (needs the FocusTools import).